README.md

```diff
@@ -10,11 +10,11 @@ home page: http://ab-initio.mit.edu/nlopt
 
 It is compiled and installed with CMake build system:
 
-    cmake .
+    cmake -DCMAKE_INSTALL_PREFIX=~/.local .
```

Comment:
Interesting, I didn't know that about Python.  Unfortunately, it is still not in the standard compiler search path so you will need `-I` and `-L` flags to compile C code with nlopt; maybe a note to that effect would be useful.

(`/usr/local` is the GNU standard default prefix for `make install` scripts, because it is the standard system-wide install location for things that are compiled from source.  Of course, it assumes you can do `sudo make install`)